### DISCUSSION TITLE
Model floating while rendering on canvas

### DISCUSSION THREAD

**Discussion by shreyasmulay2305** [google/model-viewer#4057] [Q&A]
Hello Team, 

I have a model to be rendered on Canvas using model-viewer. Somehow the model is kind of floated in air, if you see the attached image, the shadow and model has a gap.

Can someone help me how to fix this? @elalish @cdata 

Sorry if this is a repeated question, but I am unable to fix this issue. 
![image](https://user-images.githubusercontent.com/123137600/213608307-07e730af-e1cf-4606-9bb0-ffe19242b43e.png)

**Comment by elalish** [comment]
The problem is the tail. Our shadow is set at the bottom of the model's bounding box (since the origin position tends to be unreliable). If you bend the tail up, the shadow will rise to meet the paws. See some related discussion in #3985.

**Reply by shreyasmulay2305** [reply]
Thanks a lot @elalish for your explanation and prompt reply. 

What if I cannot do anything with the model and need to somehow do changes with model-viewer to make the paws go down. Do model-viewer provide any functionality to adjust the bounding box/height according to the model ?

**Reply by elalish** [reply]
Not currently, but that's the feature #3985 is asking for. Contributions welcome!
